src/containers/offer-details/offer-carousel/OfferCarousel.tsx

```diff
@@ -55,9 +55,12 @@ const OfferCarousel = ({ offer }: OfferCarouselProps) => {
     [offer]
   )
 
-  const { response } = useAxios<GetOffersResponse>({
-    service: getOffers,
-    defaultResponse
+  const { data: response } = useQuery<GetOffersResponse>({
+    queryKey: ['offers', offer],
```

Comment:
Don't use duplicate keys for useQuery. If this key is used somewhere else, it can break a cashing behavior.
```suggestion
    queryKey: ['suggested-offers', offer],
```

Reply:
Fixed.